javascript/example_code_v3/lambda/tutorial/slotassets/lambda-function-setup.js

```diff
@@ -0,0 +1,44 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+
+ABOUT THIS NODE.JS EXAMPLE: This example works with Version 3 (V3) of the AWS JavaScript SDK,
+which is scheduled for release later in 2020. The prerelease version of the SDK is available
+at https://github.com/aws/aws-sdk-js-v3. The 'SDK for JavaScript Developer Guide' for V3 is also
+scheduled for release later in 2020, and the topic containing this example will be hosted at
+https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/using-lambda-function-prep.html.
+Purpose:
+    lambda-function-setup.test.js demostrates how to create a lambda function.
```

Comment:
Should this be "Purpose: lambda-function-setup.js..."?
Also ..."how to create **an AWS Lambda** function